forem/forem
# Create Settings::SMTP

## What type of PR is this? (check all applicable)

## Description
Officially adding SMTP settings into Settings (formerly SiteConfig) so Forem admins can change SMTP credentials without fiddling with hosting level configuration.

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
1. Navigate to http://localhost:3000/admin/config as an admin.
2. Try to update any and all fields under the SMTP settings category.
3. Things should save properly.
4. Assuming you don't already have SMTP setup prior, if you try signing up a new user via email sign up (which you may have to manually enable), you should be asked to confirm your email.

## Added tests?
- [x] No, and this is why: extending existing feature

## [Forem core team only] How will this change be communicated?
- [x] This change does not need to be communicated, yet

## [optional] Are there any post deployment tasks we need to perform?
Closing monitor errors and sendgrid
